mongodbatlas/data_source_mongodbatlas_online_archive.go

```diff
@@ -83,6 +83,10 @@ func schemaOnlineArchive() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"collection_type": {
```

Comment:
are we missing similar updates to data source mongodbatlas online archive**s**? https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/online_archives